### ISSUE TITLE
client cannot start: input/output error

### ISSUE THREAD

**Issue by sveatlo** [owulveryck/goMarkableStream#7]
After starting the client, I get
```
# RK_SERVER_ADDR=10.11.99.1:2000 ./goMarkableClient
2021/03/25 14:15:31 listening on :8080, registered /video
2021/03/25 14:15:31 Error when calling GetImage: rpc error: code = Unknown desc = read /proc/3053/mem: input/output error
```

On the server side, there is no more details
```
2021/03/25 13:12:48 listening on tcp :2000
2021/03/25 13:14:17 New connection from:  10.11.99.2:48004
```

**Comment by owulveryck** [comment]
It is probably due to firmware 2.6
I will update my remarKable and investigate

**Comment by owulveryck** [comment]
@sveatlo, can you try with:

`RK_FB_ADDRESS=3512746 ./goMarkableStreamServer.arm /proc/$(pidof xochitl)/mem`

please

**Comment by owulveryck** [comment]
It should be solved in release [0.2.1](https://github.com/owulveryck/goMarkableStream/releases/tag/v0.2.1)

**Comment by sveatlo** [comment]
works great. amazing project. thank you very much.
